front/src/components/Msg/Msg.tsx

```diff
@@ -0,0 +1,63 @@
+import styled from 'styled-components';
+import theme from '../../utils/theme';
+import mock from '../../mockdata.json'; // temporary
+
+interface MsgProps {
+  elems: [string, 'Empty' | 'Not', string, string];
+}
```

Comment:
이렇게 사용하는게 가독성이 좋지는 않은느김
```suggestion
interface MsgProps {
  isInput: boolean;
  from: string;
  to: string;
  msg: string;
}
```
요런식으로 쓰는게 좋지않을까

Reply:
우옹!! 반영하겠슴다 감사합니다🙇